test/redump-blockchain.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/zsh
+
+LISK_CORE_PATH="../../lisk-service/"
+NODE_VERSION="v12.22.7"
+
+cd "$LISK_CORE_PATH/jenkins/redis"
+
+source ~/.bash_profile
+pm2 stop ../../ecosystem.core3.config.js
+
+make down
+cd ../mariadb
+make down
+
+LISK_CORE_PID=$(pgrep -u $USER lisk-core);
```

Comment:
Shouldn't pm2 be used? (Won't pm2 restart core if it's killed?)

Reply:
nope I don't think pm2 kills lisk-core if it is stopped or killed @fchavant